hudi-common/src/main/resources/hbase-site.xml

```diff
@@ -1699,13 +1699,6 @@ possible configurations would overwhelm and obscure the important.
       Implementation of the status publication with a multicast message.
     </description>
   </property>
-  <property>
-    <name>hbase.status.listener.class</name>
-    <value>org.apache.hadoop.hbase.client.ClusterStatusListener$MulticastListener</value>
```

Comment:
@xicm could you rename the class here with the shaded pattern, instead of removing the config?

Reply:
@yihua If we rename the class with the shaded name, there will be a ClassNotFoundException when referencing hudi-common.

Reply:
I mean we use shaded name with bundle jar only. If the dependence we use is hudi-common and the listener class comes from original hbase-client, in this case we will get an exception.